packages/form-js-editor/package.json

```diff
@@ -44,6 +44,7 @@
   "dependencies": {
     "@bpmn-io/form-js-viewer": "^0.9.9",
     "array-move": "^3.0.1",
+    "big.js": "^6.2.1",
```

Comment:
We should not forget to add this to [external dependencies](https://github.com/bpmn-io/form-js/blob/develop/packages/form-js-editor/rollup.config.js#L56), so we do not bundle strictly inside.

Reply:
Did that in its own commit.